WTG.Analyzers/Analyzers/Whitespace/WhitespaceCodeFixProvider.cs

```diff
@@ -126,10 +127,43 @@ static SyntaxToken RemoveTrivia(SyntaxToken token, SyntaxTrivia trivia)
 
 		static SyntaxTrivia RewriteIndentingUsingTabs(SyntaxTrivia originalTrivia, SyntaxTrivia targetTrivia)
 		{
-			return SyntaxFactory.Whitespace(StringFromIndentColumn(CalculateColumn(originalTrivia.ToString())));
+			switch (originalTrivia.Kind())
+			{
+				case SyntaxKind.WhitespaceTrivia:
+					return ModifyWhitespaceTrivia(originalTrivia);
+
+				case SyntaxKind.DocumentationCommentExteriorTrivia:
+					return ModifyDocumentationCommentExteriorTrivia(originalTrivia);
+
+				default:
+					return targetTrivia;
+			}
+
+			SyntaxTrivia ModifyWhitespaceTrivia(SyntaxTrivia trivia)
+			{
+				var (column, _) = CalculateColumn(trivia.ToString());
+				return SyntaxFactory.Whitespace(StringFromIndentColumn(column));
+			}
+
+			SyntaxTrivia ModifyDocumentationCommentExteriorTrivia(SyntaxTrivia trivia)
+			{
+				var originalText = trivia.ToString();
+				var (column, length) = CalculateColumn(originalText);
+
+				if (length == originalText.Length)
+				{
+					return SyntaxFactory.DocumentationCommentExterior(StringFromIndentColumn(column));
+				}
+
+				var builder = new StringBuilder();
+				builder.Append('\t', column / AssumedTabSize);
```

Comment:
Where does this assumption come from?

Reply:
It's an existing assumption (1 tab == 4 spaces). The tab size seems to be available on the workspace options; but I don't think I can access that from here.